doc/040_backup.rst

```diff
@@ -225,7 +225,7 @@ the exclude options are:
 
 -  ``--exclude`` Specified one or more times to exclude one or more items
 -  ``--iexclude`` Same as ``--exclude`` but ignores the case of paths
--  ``--exclude-caches`` Specified once to exclude folders containing `this special file <https://bford.info/cachedir/>`__
+-  ``--exclude-caches`` Specified once to exclude folders according to `this special file <https://bford.info/cachedir/>`_, files other than ``CACHEDIR.TAG`` and all subfolders are excluded.
```

Comment:
I'd like to avoid the double negation regarding "CACHEDIR.TAG". What about
```
Specified once to exclude a folder's content if it contains `this special file <https://bford.info/cachedir/>`__, but keep ``CACHEDIR.TAG``.
```
That takes the first part of the description of `--exclude-if-present` that is internally used to implement `--exclude-caches`.

Reply:
I've tweaked my suggestion slightly to make it clear that "this special file" == CACHEDIR.TAG.